amplify/cli.json

```diff
@@ -1,62 +1,62 @@
-{
-  "features": {
-    "graphqltransformer": {
-      "addmissingownerfields": true,
-      "improvepluralization": false,
-      "validatetypenamereservedwords": true,
-      "useexperimentalpipelinedtransformer": true,
-      "enableiterativegsiupdates": true,
-      "secondarykeyasgsi": true,
-      "skipoverridemutationinputtypes": true,
-      "transformerversion": 2,
-      "suppressschemamigrationprompt": true,
-      "securityenhancementnotification": false,
-      "showfieldauthnotification": false,
-      "usesubusernamefordefaultidentityclaim": true,
-      "usefieldnameforprimarykeyconnectionfield": false,
-      "enableautoindexquerynames": false,
-      "respectprimarykeyattributesonconnectionfield": false,
-      "shoulddeepmergedirectiveconfigdefaults": false,
-      "populateownerfieldforstaticgroupauth": false
-    },
-    "frontend-ios": {
-      "enablexcodeintegration": true
-    },
-    "auth": {
-      "enablecaseinsensitivity": true,
-      "useinclusiveterminology": true,
-      "breakcirculardependency": true,
-      "forcealiasattributes": false,
-      "useenabledmfas": true
-    },
-    "codegen": {
-      "useappsyncmodelgenplugin": true,
-      "usedocsgeneratorplugin": true,
-      "usetypesgeneratorplugin": true,
-      "cleangeneratedmodelsdirectory": true,
-      "retaincasestyle": true,
-      "addtimestampfields": true,
-      "handlelistnullabilitytransparently": true,
-      "emitauthprovider": true,
-      "generateindexrules": true,
-      "enabledartnullsafety": true
-    },
-    "appsync": {
-      "generategraphqlpermissions": true
-    },
-    "latestregionsupport": {
-      "pinpoint": 1,
-      "translate": 1,
-      "transcribe": 1,
-      "rekognition": 1,
-      "textract": 1,
-      "comprehend": 1
-    },
-    "project": {
-      "overrides": true
-    }
-  },
-  "debug": {
-    "shareProjectConfig": false
-  }
+{

```

Comment:
I cannot se what is changed in this file. We should probably use the same format for whitespace/tabs to avoid git diffs when there is actually no diff

Reply:
Yeah, was done by the amplify cli, but we can make an editor config to unify the editor options